public/data/vivek.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Vivek Kumar Singh",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/vivekthedev.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vivekthdev",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vivekthedev",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Won Hackathon",
+      "date": "October 2021",
+      "icon": "Globe",
```

Comment:
```suggestion
      "icon": "globe",
```

Reply:
This can be kept as-is, it's fine. 🙂

Reply:
@Panquesito7 Yes, but for formatting purposes, it's better in lowercase, coz the rest of the icons are in lower-case